### PR TITLE
Masthead doc

### PR DESCRIPTION
**Pull request by Sylphias**
- Add documentation for Masthead component

### CODE REVIEW

**Comment by Sylphias** [comment]
- Also added breadcrumb documentation
- Refactored breadcrumb component as well

File: src/components/Breadcrumb.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ class Breadcrumb extends Component {
       <nav className={this.getNavClassName()} aria-label="breadcrumbs">
         <ul>
           {items.map((value, index) => {
+            console.log(value)
```
Comment (weijunyu) [inline src/components/Breadcrumb.js]: Left this in

**Comment by Sylphias** [comment]
Okay Removed console logs from the following files
- Breadcrumb
- Main Nav

File: src/components/Breadcrumb.js (hunk 1 above)
Comment (Sylphias) [reply src/components/Breadcrumb.js]: Okay Removed console logs from the following files

Breadcrumb
Main Nav

**Pull request merged**